asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js

```diff
@@ -130,6 +140,12 @@ const ExportPage = (props) => {
     queryClient.resetQueries("fetchDatasetWriter");
   };
 
+  const [dialogOpen, setDialogOpen] = useState(false);
+
+  const handleOpen = () => {
+    setDialogOpen(true);
+  };
```

Comment:
This function is available for open but not for close. It's more consistent to make this into a toggle function.

Reply:
done